src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: normal;
+  color: #3d4e61;
+}
+
+.wrap-big {
+  height: 70px;
+}
+
+.wrap-small {
+  height: 42px;
```

Comment:
Pay attention to this requirement
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/637889a8-72c9-4994-a703-df3b1e25cd01)